src/display/editor/stamp.js

```diff
@@ -425,7 +426,9 @@ class StampEditor extends AnnotationEditor {
 
     this._uiManager.enableWaiting(false);
     const canvas = (this.#canvas = document.createElement("canvas"));
-    div.append(canvas);
+    this.addContainer(canvas);
+    canvas.setAttribute("role", "img");
```

Comment:
Maybe swap the order of these, since it seems theoretically more efficient to set this attribute *before* adding the element to the DOM?
```suggestion
    canvas.setAttribute("role", "img");
    this.addContainer(canvas);
```